README.md

```diff
@@ -30,18 +30,20 @@ to your `pom.xml` (without the dots):
     <dependency>
             <groupId>org.eclipse.californium</groupId>
             <artifactId>californium-core</artifactId>
-            <version>1.0.4</version>
+            <version>1.0.6</version>
     </dependency>
     ...
   </dependencies>
   ...
 ```
 
-If you want to give the most recent milestone a try, use version `2.0.0-M2` instead.
+If you want to give the most recent milestone a try, use version `2.0.0-M6` instead.
+
+Please Note: in the meantime, a lot of new bugfixes and improvements has been made in branch 2.0.x, which had affected the API and could not be back-ported to the other branches. Even if 2.0.x is still under development, we recommend to focus on using the 2.0.x branch, especially for larger changes.
 
 ##### Current Master Builds
 
-You can also be bold and try out the most recent build from `master`.
+You can also be bold and try out the most recent build from `master` or `2.0.x`.
```

Comment:
I think we shouldn't encourage people to use `master` at all since we do not plan to do a 1.1 release anyway, right?
FMPOV we should replace `master` with `2.0.x` and adapt the heading to `Nightly Builds`

Reply:
Maybe we even better switch master to 2.0.x.